### PR TITLE
No Case: RLLib 1.3.0 fixed the DQN convergence issue per master branc…

### PR DESCRIPTION
**Pull request by RuofanKong**
…h fix.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/agents/dqn/dqn_tf_policy.py [hunk 1]
```diff
@@ -383,41 +384,13 @@ def compute_q_values(policy: Policy,
     return value, logits, dist, state
 
 
-def _adjust_nstep(n_step, gamma, obs, actions, rewards, new_obs, dones):
```
Comment (Kiko-Aumond) [inline rllib/agents/dqn/dqn_tf_policy.py:386]: Can you document all input variables in this method?

**Review by Kiko-Aumond — APPROVED** [review]

File: rllib/agents/dqn/dqn_tf_policy.py (hunk 1 above)
Comment (RuofanKong) [reply rllib/agents/dqn/dqn_tf_policy.py:386]: > Can you document all input variables in this method?

This method is moved to the one in `rllib/evaluation/postprocessing.py` and I think Sven added the doc string for the method.

Comment (Kiko-Aumond) [reply rllib/agents/dqn/dqn_tf_policy.py:386]: It'd be useful to have documentation and type annotations for those parameters.

Comment (RuofanKong) [reply rllib/agents/dqn/dqn_tf_policy.py:386]: > It'd be useful to have documentation and type annotations for those parameters.

Yes, the type annotation is included there.

Comment (RuofanKong) [reply rllib/agents/dqn/dqn_tf_policy.py:386]: see https://github.com/BonsaiAI/ray/pull/93/files#diff-68633652bfaca9ea068b0fbd5210cb01032ee46fd770487b542e8b1722cfc80bR19

**Pull request merged**